config/routes.js

```diff
@@ -32,10 +32,7 @@ module.exports.routes = {
   *                                                                          *
   ***************************************************************************/
 
-  '/*': function log (req, res, next) {
-    sails.log.info(new Date(), req.method, req.url);
-    next();
-  },
+  '/*': { policy: 'angularRouting', skipAssets: true },
```

Comment:
Now i'm thinking that `skipAssets` and `skipRegex` are not being applied because of the policy and that maybe I should use a controller to server the page.